my_gazebo_concert/src/solutions/concert.launch

```diff
@@ -0,0 +1,17 @@
+<launch>
```

Comment:
This file can be deleted, and you can invoke the same roslaunch command but pass the argument `gazebo_services:=my_gazebo_concert/gazebo.services`